### ISSUE TITLE
enable512e can't handle False value

### ISSUE THREAD

**Issue by scaleoutsean** [NetApp/terraform-provider-netapp-elementsw#6]
`true` works, `false` doesn't. 

```
  # elementsw_volume.storage-volume[1] will be created
  + resource "elementsw_volume" "storage-volume" {
      + account    = (known after apply)
      + burst_iops = 1000
      + enable512e = false
      + id         = (known after apply)
      + iqn        = (known after apply)
      + max_iops   = 500
      + min_iops   = 100
      + name       = "netapp-trident-vol-1"
      + total_size = 107374182400
    }
```

Result:

```
elementsw_volume.storage-volume[1]: Creating...
2020/06/26 09:00:43 [ERROR] <root>: eval: *terraform.EvalApplyPost, err: enable512e argument is required
2020/06/26 09:00:43 [ERROR] <root>: eval: *terraform.EvalSequence, err: enable512e argument is required

Error: enable512e argument is required

  on main.tf line 9, in resource "elementsw_volume" "storage-volume":
   9: resource "elementsw_volume" storage-volume {
```

[This part](https://github.com/NetApp/terraform-provider-netapp-elementsw/blob/b43da382fd01353b8e915438ea6a8077517face4/elementsw/resource_elementsw_volume.go#L136) could be what causes problems, maybe thanks to changes in TF 0.12 (assuming it worked before).

```
terraform version
Terraform v0.12.26
+ provider.netapp-elementsw (unversioned)
```

**Comment by lonico** [comment]
yes, it;s an issue with:
```
	if v, ok := d.GetOk("enable512e"); ok {
		volume.Enable512E = v.(bool)
	} else {
		return fmt.Errorf("enable512e argument is required")
	}
```

**Comment by lonico** [comment]
fixed with 54b3e9cd586aa0512dd3551182ca202ff9c975ef
